### DISCUSSION TITLE
Complete the lyrics 🎤

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#698] [Galaxy Trivia] [answered]
```
You'll take my ________, but I'll take yours too
You'll fire your musket, but I'll ________ you through
So when you're waiting for the next ________
You'd better ________, there's no turning back

The bugle sounds, the ________ begins
But on this battlefield, no one ________
The ________ of acrid smoke and horses' breath
As I plunge on into certain ________

Oh oh oh oh oh oh oh oh oh
Oh oh oh oh oh oh oh oh oh
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment, marked as answer]
You'll take my **LIFE**, but I'll take yours too
You'll fire your musket, but I'll **RUN** you through
So when you're waiting for the next **ATTACK**
You'd better **STAND**, there's no turning back

The bugle sounds, the **CHARGE** begins
But on this battlefield, no one **WINS**
The **SMELL** of acrid smoke and horses' breath
As I plunge on into certain **DEATH**

Oh oh oh oh oh oh oh oh oh
Oh oh oh oh oh oh oh oh oh

Iron Maiden's The Trooper was written by bassist and founding member **Steve Harris**.
